cmd/ci-secret-bootstrap/main.go

```diff
@@ -777,6 +813,23 @@ func main() {
 	})
 	defer logrus.Exit(0)
 
+	if o.validateOnly {
```

Comment:
This isn't fixed, `o.ValidateOptions` will still require kubeconfigs even though they are not needed.
/hold

Reply:
Yea, I am on it. Thats why I kept this PR on hold

Reply:
updated. Please check.